ProteoformSuiteGUI/ExperimentExperimentComparison.cs

```diff
@@ -46,15 +46,19 @@ public void run_the_gamut()
         {
             this.Cursor = Cursors.WaitCursor;
             Lollipop.make_ee_relationships();
-            ((ProteoformSweet)MdiParent).proteoformFamilies.ClearListsAndTables();
-            Parallel.Invoke
-            (
-                () => this.FillTablesAndCharts(),
-                () => Lollipop.proteoform_community.construct_families()
-            );
-            ((ProteoformSweet)this.MdiParent).proteoformFamilies.initialize_settings();
-            ((ProteoformSweet)this.MdiParent).proteoformFamilies.fill_proteoform_families("");
-            ((ProteoformSweet)this.MdiParent).proteoformFamilies.update_figures_of_merit();
+            //slow for labelfree - need to manually check peaks and raise peak threshold or too many peaks
+            if (Lollipop.neucode_labeled)
```

Comment:
Thanks for pointing out these performance issues. Let's try to pick the EE cutoff in a smart way to avoid these issues. With that, I think this performance issue would be resolved and also go quicker with the parallelization.